### PR TITLE
DEVELOPER-2580 Fixed BRMS/overview video layout

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

File: _layouts/product-overview.html.slim [hunk 1]
```diff
@@ -42,3 +42,13 @@ br
     = page._detail
     .large-16
       = video_player(video(page.product.intro_video_2)) if page.product.intro_video_2
+
+  - if page.product.id = 'brms'
```
Comment (paulrobinson) [inline _layouts/product-overview.html.slim]: Can this be done without putting product specific content in the template?

**Comment by adelasofia** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Comment by RubyGemTSL** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Pull request closed without merging**